### DISCUSSION TITLE
Is it possible to expose k3s api-server as HTTP endpoint (without certificates)?

### DISCUSSION THREAD

**Discussion by senthilprabhut** [k3s-io/k3s#9072] [Q&A] [answered]
Hi, I'm trying to run k3s docker along with couple other services (inside a K8S cluster) and one of those services have to make a k8s api call
Currently, K3S is generating a self-signed certificate. So the calling service has to ignore the self-signed certificate check.

Is there a possibility to expose the k3s api-server as HTTP so that other services can call into it without the overhead of certificate validation.

**Comment by brandond** [comment, marked as answer]
What do you mean by "the k3s service". Are you referring to the Kubernetes apiserver on port 6443? In that case no, there is no way to disable TLS.
